course6/toyapp-adam/displaylib/scene.cc

```diff
@@ -0,0 +1,67 @@
+#include "scene.h"
+
+#include "surface.h"
+#include "util/keyboard.h"
+#include "util/mouse.h"
+#include <cmath>
+
+namespace Display {
+
+Scene::Scene(Keyboard& keyboard, Mouse& mouse, Surface3D& surface)
```

Comment:
This _is_ your app. This shouldn't be part of the display library

Reply:
You're right.